modin/pandas/test/test_io.py

```diff
@@ -1643,6 +1643,26 @@ def test_read_parquet_2462(self):
 
             df_equals(test_df, read_df)
 
+    @pytest.mark.xfail(
+        condition="config.getoption('--simulate-cloud').lower() != 'off'",
```

Comment:
Do we need this?

Reply:
They were on the other tests, so I figured I would add it here as well.